src/helper.js

```diff
@@ -113,6 +113,12 @@ function setHome(newPath) {
 	} else {
 		this.config.theme = `themes/${this.config.theme}.css`;
 	}
+
+	if (this.config.leaveMessage.includes("thelounge.github.io")) {
```

Comment:
I don't really see a need in doing this.

Reply:
I'd like to keep this for a couple version. It doesn't cost anything, and it might be useful for whoever started to use The Lounge with versions 2.5 or 2.6, or copied this in their config. There will be a redirect in the URL, but that might definitely help.

Reply:
@McInkay what do you think?

Reply:
I think it would make more sense that if the leave message is the default one, changing it, because they clearly don't care about what the default leave message is.

But I don't think it's really necessary, it redirects, so ...  ¯\\\_(ツ)_/¯

Reply:
We can't change the values in the config file ourselves after the first start (i.e. when the file has been copied over). Did that in the past and there were permission issues, I don't want to break any more things 😅

I can remove this if you don't like it, but otherwise we can keep it in v3.0 and remove in v3.1. That will be a reminder for people to change it in their config.

Reply:
I think asking to change Lounge url will just push most people to either not care, or change it to something completely different :)

Reply:
Ok, I removed it.